### PR TITLE
correct typos, add small amount of content to CoC and Handbook

### PR DESCRIPTION
**Pull request by beccastevens**

### CODE REVIEW

File: content/code-of-conduct.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ do not tolerate harassment of participants in any form.
 This code of conduct applies to all STL Tech spaces, including the STL Tech
 Slack and Github Repositories, both online and off. Anyone who violates this
 code of conduct may be sanctioned or expelled from these spaces at the
-discretion of the the incident response team.
+discretion of the the incident response team. **Please note that STL Tech spaces are not public spaces** and that interpretation of this Code of Conduct in matters of sanctioning or expelling participants in violation of same is at the sole discretion of the incident response team.
```
Comment (Hellmark) [inline content/code-of-conduct.md]: Should the note for this be on the same line, or a new line?

Comment (beccastevens) [reply content/code-of-conduct.md]: oh good point, might be better on a new line

**Review by Hellmark — APPROVED** [review]
Overall, it is looking good to me.

**Review by booch — APPROVED** [review]
Looks good. Honestly, I thought some of those changes had already been made.

**Pull request merged**